### ISSUE TITLE
Small changes in the Add link dialogue

### ISSUE THREAD

**Issue by rdstern** [IDEMSInternational/R-Instat#6350]
The Add Key dialogue now provides a small summary in the output Window.  

Here there are 2 small items

a) Suggest a name as we do in other dialogues - `link` would be fine
b) As with the Add Key, it would be good to provide some output.  Possibly just 
`name`  variable in `name of data frame1` linked to `name `  variable in `name of data frame2 `
For example `station variable in data frame:germany_data linked to station variable in data frame germany_metadata` 

![image](https://user-images.githubusercontent.com/11226469/112725627-1d140780-8f11-11eb-823c-601b376069f9.png)

@dannyparsons to check quickly.  It would slightly simplify the import data from Germany video!

**Comment by rdstern** [comment]
@Ivanluv the dialogue now does give some output.  So the only task is to add the default name of `link`.

**Comment by shadrackkibet** [comment]
What if there is an existing link called "link"? I would like @dannyparsons to confirm this change. I think for some reason we left this without the default which I don't remember. I think we want to have automatic indexing e.g link1, link2 etc also for keys?

**Comment by rdstern** [comment]
Yes, if it is like our norm in other dialogues, then it would be called `link` and if link exists, then `link1`, `link2`, etc.
